### ISSUE TITLE
New align shortcuts

### ISSUE THREAD

**Issue by RONNYKHALIL** [coolzilj/ComfyUI-LJNodes#8]
I really love this new feature. You're making ComfyUI closer to how smooth PureRef!

However "Alt + W/S/A/D to align selected nodes." doesn't seem to work for me. I'm on a Mac, but the majority of my comfyui work is done in Runpod and Paperspace. I tried both of them plus my local mac install and none seem to work. The original implemented shortcuts (last night!) did work. Any ideas?

**Comment by coolzilj** [comment]
Did you update to the latest version?  
I haven't tested it on Mac or Linux, though.

**Comment by RONNYKHALIL** [comment]
Yea latest

**Comment by makeitrad** [comment]
Yes this is a problem for Mac users. the Alt key is used on a system level. Alt-W closes windows, Alt-H Hides the current window. Mac users need the ability to remap to ctrl or option.

**Comment by HaoJingang** [comment]
I am in China, and in many of our applications, Alt+A is used for taking screenshots. Of course, I know that the WASD four buttons are as well-known as the front, back, left, right buttons in games. Due to the conflict, I would like to change it to Ctrl/Alt+L/R/T/B (L=left, R=right,T=Top,B=Bottom)。

In addition, in Blender, 7 can be used to switch top views, 1 and 3 can switch left and right views, and 2 is the bottom view.

Based on my experience using these software, the numbers 7123 are the fastest. If the author cannot set specific shortcut keys, I hope they can be customized because everyone has different habits formed by the software they have used.

**Comment by m21-cerutti** [comment]
Would  be the same for other users with not QWERTY convention, would be great with https://github.com/coolzilj/ComfyUI-LJNodes/issues/10
